akka-persistence/src/test/scala/akka/persistence/PersistentActorSpec.scala

```diff
@@ -248,6 +248,14 @@ object PersistentActorSpec {
         persistAsync(event) { evt ⇒ sender() ! s"${evt.data}-b-${sendMsgCounter.incrementAndGet()}" }
     }
   }
+  class PersistAllNilPersistentActor(name: String) extends ExamplePersistentActor(name) {
+
+    val receiveCommand: Receive = commonBehavior orElse {
+      case Cmd(data) ⇒
+        persistAll(Nil)(_ ⇒ ())
```

Comment:
So this loops the journal and then calls the handler? I think that is necessary for ordering.